### ISSUE TITLE
The shell-operator start command is not stopping when sent ^C.

### ISSUE THREAD

**Issue by GrahamDumpleton** [flant/shell-operator#186]
With version of ``shell-operator`` from ``flant/shell-operator:v1.0.0-beta.10``, but pulled out of that image into separate image where it is then being run from command line in the container, interrupting it with ``^C` is not resulting in it shutting down properly, even though it says it is gracefully shutting down.

The messages output are shown below, with you being able to see where ``^C`` was generated:

```
time="2020-07-05T05:41:52Z" level=info msg="Hook executed successfully" binding=kubernetes hook=monitor-service-accounts.sh operator.component=taskRunner queue=main task=HookRun

^Ctime="2020-07-05T05:41:57Z" level=info msg="Grace shutdown by 'interrupt' signal"
time="2020-07-05T05:41:57Z" level=info msg=Stop operator.component=handleEvents
```

If you subsequently send ``^C`` again, then forced shutdown works fine.

```
^Ctime="2020-07-05T05:42:12Z" level=info msg="Forced shutdown by 'interrupt' signal"
```

Using ``--log-level debug`` only results in the following extra details.

```
^Ctime="2020-07-05T05:47:50Z" level=info msg="Grace shutdown by 'interrupt' signal"
time="2020-07-05T05:47:50Z" level=info msg=Stop operator.component=handleEvents
time="2020-07-05T05:47:50Z" level=debug msg="queue main: got nil task, stop queue"

^Ctime="2020-07-05T05:47:57Z" level=info msg="Forced shutdown by 'interrupt' signal"
```

**Comment by diafour** [comment]
Hello! There was a problem with "Grace shutdown" mode: it stops a queue but not exits a process and continues to handle new Kubernetes events. There is a fix for this in PR #176. Now it stops handling Kubernetes events, Schedule events, stops all queues and exits immediately, or after 10 seconds if there is a running hook. It will be released in version `beta.11` this week I think.

**Comment by diafour** [comment]
@GrahamDumpleton The fix is in beta.11 https://github.com/flant/shell-operator/releases/tag/v1.0.0-beta.11.
